libs/feature/editor/src/lib/services/editor.service.spec.ts

```diff
@@ -15,7 +15,7 @@ const SAMPLE_RECORD: CatalogRecord = datasetRecordsFixture()[0]
 class RecordsRepositoryMock {
   openRecordForEdition = jest.fn(() =>
     of([
-      { ...SAMPLE_RECORD, recordUpdated: new Date() },
+      { ...SAMPLE_RECORD, resourceUpdated: new Date() },
```

Comment:
Yes, it was failing, I wouldn't have noticed it otherwise :smile: 
I thought the test was just about changing any value in the form and test that it was recorded, TBH I could've used any other field but this one came to mind because it was closer... But maybe I misunderstood the aim of this test?

Reply:
I understand the test like that:
we save the record and we test if after saving the record the `updatedRecord` gets updated (to now) and therefore is different from the previous value that was in `updatedRecord`.

But I don't get why it is failing. I will have a look.

Reply:
I think it is caused by the deletion  of the `RECORD_UPDATED_FIELD` in the `fields.config.ts`. When you look at the `beforeEach`of `saveRecord` in the test, you will see that for saving the record the `DEFAULT_CONFIGURATION` is being used.
So it might be a mistake to remove the `RECORD_UPDATED_FIELD` and maybe even the uuid field from the config. Maybe we just need to remove it from the UI?

Reply:
I think you're right on that. The issue is that I used to have them kept in `fields.config.ts` and the editor was still displaying a space + title for both fields, because it iterates from that file (I think), which we do not which for :slightly_frowning_face:

Reply:
Let's talk about this tomorrow in the daily.

Reply:
Which changes did you make? :)

Because it's not just the white space though, it's also their titles, here I added them back : 
![image](https://github.com/user-attachments/assets/c9aa72f4-e791-4b3b-9b5a-e749d457ca73)

Reply:
It's this commit: https://github.com/geonetwork/geonetwork-ui/pull/1032/commits/77b7f49fe33fdc4309c0723218ae36f04dda2a01

the changes we made to:
- libs/feature/editor/src/lib/fields.config.ts
- libs/feature/editor/src/lib/models/editor-config.model.ts
- libs/feature/editor/src/lib/+state/editor.actions.ts

With this in place we could set the field visibility.

But now I am uncertain... Can we not simply set the `hidden` property of those fields to `true`?

Reply:
Yes, I think this could work! But since those 2 fields will never be used again I don't know if it's good practice to keep unused code...

Reply:
```
export const RECORD_UNIQUE_IDENTIFIER_FIELD: EditorField = {
  model: 'uniqueIdentifier',
  formFieldConfig: {
    labelKey: marker('editor.record.form.field.uniqueIdentifier'),
    locked: true,
  },
  hidden: true
}
```

Reply:
Actually I think the fields are still in use (that's why the test failed) but they are not visible to the user.

Reply:
That makes sense!